### ISSUE TITLE
Still supported? Can't get it to work with XDM and webpack

### ISSUE THREAD

**Issue by davidfloegel** [remcohaszing/remark-mdx-code-meta#2]
Hi there,

Thanks for the package!

I'm currently trying to make this work with a custom react webpack setup to expose meta data on code blocks.

This is what my webpack config looks like:

```
const codeMeta = require('remark-mdx-code-meta');

[...]
      {
        test: /\.mdx?$/,
        use: [
          {
            loader: 'xdm/webpack.cjs',
            options: {
              providerImportSource: '@mdx-js/react',
              remarkPlugins: [codeMeta.remarkMdxCodeMeta],
            },
          },
        ],
      },
[...]
```

While it compiles, I can't seem to get the actual plugin to work.

This is my React file:

```
import React from 'react'
import { MDXProvider } from '@mdx-js/react';
import Test from './test.mdx'

const components = {
  code: (p) => {
    console.log(p);
    return <div />
  }
}

export const Preview = () => (
  <MDXProvider components={components}>
    <Test />
  </MDXProvider>
)
```

and my `test.mdx` looks like:

````
# Test

```js render
<button>Test</button>
```
````

I'm expecting to see the `render` attribute in the props passed to the `code` element but no luck.

Can you tell me if I'm missing anything?

Thanks a lot!

**Comment by remcohaszing** [comment]
It should still work with xdm.

A markdown code block is transformed to an HTML `<pre>` element containing a `<code>` element. The props are passed to the `<pre>` element, meaning you should use that in your custom components.

TL;DR:

```diff
  const components = {
-   code: (p) => {
+   pre: (p) => {
      console.log(p);
      return <div />
    }
  }
```

Also [MDX 2 was released 2 days](https://mdxjs.com/blog/v2/) ago. This basically brings parity with xdm. Xdm is slowly going to get phased out. So I highly suggest using `@mdx-js/loader` instead. It should work as a drop in replacement.
